index.html

```diff
@@ -1 +1,50 @@
+<html><head>
```

Comment:
REREAD the html documentation, head isn't the "header" of the front page, you put that in a <body>.